internal/controller/driver_controller.go

```diff
@@ -1053,6 +1083,10 @@ func (r *driverReconcile) isNfsDriver() bool {
 	return r.driverType == NfsDriverType
 }
 
+func (r *driverReconcile) logRotation() bool {
+	return r.driver.Spec.Log.Rotation != nil
+}
```

Comment:
See the previous comment about pre-calculating the argument.
It should make this function redundant